src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -46,18 +53,18 @@ public HostController(WebScriptHostManager scriptHostManager, WebHostSettings we
         [HttpGet]
         [Route("admin/host/status")]
         [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
-        [EnableDebugMode]
-        public IActionResult GetHostStatus()
+        [TypeFilter(typeof(EnableDebugModeFilter))]
+        public async Task<IActionResult> GetHostStatus([FromServices] IScriptHostManager scriptHostManager, [FromServices] IHostIdProvider hostIdProvider)
```

Comment:
These have to be [FromServices] method params rather than ctor params because of specialization lifetime concerns?